lib/check-port.js

```diff
@@ -0,0 +1,33 @@
+const tcpPortUsed = require('tcp-port-used');
+const chalk = require('chalk');
+
+function checkPort(port, content) {
+  let numAttempts = 0;
```

Comment:
You are trying to call checkPort recursively but due to variable scoping this `numAttempts` will always be 0 and never got incremented

Reply:
oh! yes, fixing 😄